helper/tests/helper-tests-110/src/test/java/com/linkedin/avroutil1/compatibility/avro110/Avro110FieldBuilderTest.java

```diff
@@ -159,6 +160,17 @@ public void testNullDefaultForBoolField() throws Exception {
     }
   }
 
+  @Test
+  public void testArrayOfEnumDefaultValue() throws IOException {
+    Schema schema = Schema.parse(TestUtil.load("FieldWithArrayOfEnumDefaultValue.avsc"));
+    Schema.Field field = schema.getField("arrayOfEnum");
+    Object defaultValue = AvroCompatibilityHelper.getGenericDefaultValue(field);
+    FieldBuilder builder = AvroCompatibilityHelper.newField(field);
+    builder.setDefault(defaultValue);
+    Schema.Field resField = builder.build();
+    Assert.assertNotNull(resField.defaultVal());
```

Comment:
Add an assertEquals test as well

Reply:
Honestly the assert isn't the point of the test. The point of the test is that it doesn't fail to `.build()`. Added a comment in the code